app/helpers/mission_control/jobs/jobs_helper.rb

```diff
@@ -20,7 +20,7 @@ def attribute_names_for_job_status(status)
     when "failed"      then [ "Error", "" ]
     when "blocked"     then [ "Queue", "Blocked by", "Block expiry" ]
     when "finished"    then [ "Queue", "Finished" ]
-    when "scheduled"   then [ "Queue", "Scheduled" ]
+    when "scheduled"   then [ "Queue", "Scheduled", ""]
```

Comment:
There's a space missing here: 
```suggestion
    when "scheduled"   then [ "Queue", "Scheduled", "" ]
```
I think Rubocop is complaining about that as well.